src/fragments/lib/push-notifications/ios/setup_push_service/setup-apns.mdx

```diff
@@ -69,44 +67,44 @@ If you already have an SSL certificate for your app, you can skip this step.
 
 Register a test device with your Apple Developer account so that you can test your app on that device. Later, you associate this test device with your provisioning profile, which allows your app to launch on your device.
 
-If you already have a registered device, you can skip this step.
+_If you already have a registered device, you can skip this step._
 
 **To add a device**
```

Comment:
The next three steps are done automatically in Xcode when the "Automatically manage signing" box is checked, and is usually the better option.

Reply:
Added a callout 👍